src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -466,13 +466,35 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
         return emails;
     }
 
+    /**
+     * Generates the feedback session closed email for both
+     * the instructors and the students with the given
+     * {@code userName} and {@code userEmail}.
+     */
     private EmailWrapper generateFeedbackSessionClosedEmail(CourseAttributes course,
             FeedbackSessionAttributes session, String userName, String userEmail) {
+
```

Comment:
If this line is reversed, there will be no blank line between the method's arguments and the rest of the method. Do we need that? According to all the other methods, we do need a blank line.